doc/modules/mod_caps.md

```diff
@@ -6,5 +6,25 @@ It is not this module's responsibility to intercept and answer disco requests ro
 ### Options
 
 This module expects two optional arguments that apply to [cache tab](https://github.com/processone/cache_tab):
-* `cache_size` (default: 1000) - the size of a cache_tab (the amount of entries) holding the information about capabilities of each user. 
-* `cache_life_time` (default: 86) - time (in seconds) after which entries will be removed
+
+#### `modules.mod_caps.cache_size`
+* **Syntax:** non-negative integer
+* **Default:** `1000`
+* **Example:** `cache_size = true`
```

Comment:
Don't think `true` is a `non-negative integer`

Reply:
you don't say